holidays/locale/en_US/LC_MESSAGES/XNSE.po

```diff
@@ -0,0 +1,107 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+#
+# National Stock Exchange of India holidays en_US localization.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Holidays 0.80\n"
+"POT-Creation-Date: 2025-08-21 16:22+0000\n"
+"PO-Revision-Date: 2025-08-21 16:22+0000\n"
+"Last-Translator: Saheel Sapovadia saheelsapovadia23@gmail.com\n"
+"Language-Team: Holidays Localization Team\n"
+"Language: en_US\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Lingva 5.0.6\n"
+"X-Source-Language: en_IN\n"
+
+#, c-format
+msgid "%s"
+msgstr "%s"
+
+#. Republic Day.
+msgid "Republic Day"
+msgstr "Republic Day"
+
+#. Dr. Baba Saheb Ambedkar Jayanti.
+msgid "Dr. Baba Saheb Ambedkar Jayanti"
+msgstr "Dr. Baba Saheb Ambedkar Jayanti"
+
+#. Maharashtra Day.
+msgid "Maharashtra Day"
+msgstr "Maharashtra Day"
+
+#. Independence Day.
+msgid "Independence Day"
+msgstr "Independence Day"
+
+#. Mahatma Gandhi Jayanti.
+msgid "Mahatma Gandhi Jayanti"
+msgstr "Mahatma Gandhi Jayanti"
+
+#. Christmas Day.
+msgid "Christmas Day"
+msgstr "Christmas Day"
+
+#. Bakri Id
+msgid "Bakri Id"
+msgstr "Bakri Id"
+
+#. Diwali Balipratipada
+msgid "Diwali Balipratipada"
+msgstr "Diwali Balipratipada"
+
+#. Diwali Laxmi Pujan
+msgid "Diwali Laxmi Pujan"
+msgstr "Diwali Laxmi Pujan"
+
+#. Dussehra
+msgid "Dussehra"
+msgstr "Dussehra"
+
+#. Ganesh Chaturthi
+msgid "Ganesh Chaturthi"
+msgstr "Ganesh Chaturthi"
+
+#. Good Friday
+msgid "Good Friday"
+msgstr "Good Friday"
+
+#. Guru Nanak Jayanti
+msgid "Guru Nanak Jayanti"
+msgstr "Guru Nanak Jayanti"
+
+#. Holi
+msgid "Holi"
+msgstr "Holi"
+
+#. Id Ul Fitr (Ramadan Eid)
+msgid "Id-Ul-Fitr (Ramadan Eid)"
+msgstr "Id-Ul-Fitr (Ramadan Eid)"
```

Comment:
`en_US` for "Id-Ul-Fitr (Ramadan Eid)", "Bakri Id", and "Muharram" should be "Eid al-Fitr", "Eid al-Adha", and "Ashura", respectively